generation/generate_gapic_bom.sh

```diff
@@ -54,5 +54,7 @@ done
 mkdir -p gapic-libraries-bom
 
 GENERATION_DIR=$(dirname -- "$0");
+perl -0pe 's/<dependencies>.*<\/dependencies>/<dependencies>\nBOM_ARTIFACT_LIST\n    <\/dependencies>/s' ${GENERATION_DIR}/../gapic-libraries-bom/pom.xml > ${GENERATION_DIR}/bom.pom.xml
```

Comment:
It's strange to use gapic libraries BOM to generate gapic libraries BOM. Why do you have remove the template?

Reply:
We've had several instances where the bom and parent pom checks started failing because changes were made without updating the templates. It's more accurate to call these "update" rather than "generate" scripts, which is really what we're doing when adding a new library. If you take that perspective, I think it makes sense to not use the templates.